packages/opentelemetry-resources/test/util/resource-assertions.ts

```diff
@@ -17,7 +17,12 @@
 import { SDK_INFO } from '@opentelemetry/core';
 import * as assert from 'assert';
 import { IResource } from '../../src/IResource';
-import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions';
+import {
+  SEMRESATTRS_TELEMETRY_SDK_LANGUAGE,
```

Comment:
As the previous names where wrong, just took this opportunity to start using the new exported values rather than re-using the namespaced version.